### PR TITLE
Publish individual module jars to Bintray

### PR DESCRIPTION
**Pull request by jframe**
## PR Description
This publishes individual module jars to Bintray under the tech.pegasys.teku.internal groupId.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -44,6 +44,28 @@ gradle.startParameter.taskNames.each {
...
+def bintrayUser = project.hasProperty('bintrayUser') ? project.property('bintrayUser') : System.getenv('BINTRAY_USER')
+def bintrayKey = project.hasProperty('bintrayApiKey') ? project.property('bintrayApiKey') : System.getenv('BINTRAY_KEY')
+
+def bintrayPackage = bintray.pkg {
+  repo = 'pegasys-repo'
+  name = 'teku'
+  userOrg = 'consensys'
+  desc = 'Java Implementation of the Ethereum 2.0 Beacon Chain'
+  licenses = ['Apache-2.0']
+  websiteUrl = 'https://github.com/PegaSysEng/artemis'
```
Comment (macfarla) [inline build.gradle]:
```suggestion
  websiteUrl = 'https://github.com/PegaSysEng/teku'
```

File: build.gradle [hunk 2]
```diff
@@ -44,6 +44,28 @@ gradle.startParameter.taskNames.each {
...
+  issueTrackerUrl = 'https://github.com/PegaSysEng/artemis/issues'
+  vcsUrl = 'https://github.com/PegaSysEng/artemis.git'
```
Comment (macfarla) [inline build.gradle]:
```suggestion
  vcsUrl = 'https://github.com/PegaSysEng/teku.git'
```

File: build.gradle [hunk 3]
```diff
@@ -44,6 +44,28 @@ gradle.startParameter.taskNames.each {
...
+def bintrayKey = project.hasProperty('bintrayApiKey') ? project.property('bintrayApiKey') : System.getenv('BINTRAY_KEY')
+
+def bintrayPackage = bintray.pkg {
+  repo = 'pegasys-repo'
+  name = 'teku'
+  userOrg = 'consensys'
+  desc = 'Java Implementation of the Ethereum 2.0 Beacon Chain'
+  licenses = ['Apache-2.0']
+  websiteUrl = 'https://github.com/PegaSysEng/artemis'
+  issueTrackerUrl = 'https://github.com/PegaSysEng/artemis/issues'
```
Comment (macfarla) [inline build.gradle]:
```suggestion
  issueTrackerUrl = 'https://github.com/PegaSysEng/teku/issues'
```

**Review by macfarla — APPROVED** [review]

File: build.gradle (hunk 1 above)
Comment (jframe) [reply build.gradle]: done

File: build.gradle (hunk 3 above)
Comment (jframe) [reply build.gradle]: done

File: build.gradle (hunk 2 above)
Comment (jframe) [reply build.gradle]: done

**Review by usmansaleem — APPROVED** [review]
looks good

**Pull request merged**